test/e2e/infra/secondary_network/secondary_network_prerequisite.sh

```diff
@@ -0,0 +1,360 @@
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+THIS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+pushd $THIS_DIR
+
+_usage="Usage: $0 [--configure-prerequisites] [--deconfigure-prerequisites]
+                  [--base-interface <interface name>] [--sriov-vf-count <count>] [--host_name]
+
+Deploy Antrea CNI and its native secondary network prerequisites on an existing/running K8s cluster control node.
+
+        --configure-prerequisites     Deploy all the pre-requisite plugins and CRDs for secondary network configuration.
+        --deconfigure-prerequisites   Remove all the pre-requisite plugins and CRDs deployed for secondary network configuration.
+	--base-interface         Network interface to create VFs.
+        --sriov-vf-count         Number of virtual functions to be created on the base network interface.
+        --host-name            	 K8s cluster control node's host name or IP address."
+
+_ssh_config_info="Prerequisite: Kubernetes(K8s) cluster is up and running at the local or remote server.
+
+Follow these steps to configure SSH information to access the kubernetes cluster control node.
+
+1) Update remote/local cluster control node information at your /home/<user>/.ssh/config
+Example:
+Host <Control-Plane-Node>
+    HostName <Control-Plane-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+
+2) Add remote/local cluster control node's public key (id_rsa.pub) to /home/<user>/.ssh/authorized_keys
+
+3) Set PubkeyAuthentication at sshd_config to yes at your remote/local cluster control node
+
+4) Esure SSH to the remote/local node works after the above configuration (ssh <control node/host name>)"
+
+function echoerr {
+        echo >&2 "$@"
+}
+
+function print_usage {
+	echoerr "$_usage"
+}
+
+function print_ssh_config_info {
+	echoerr "$_ssh_config_info"
+}
+
+function print_help {
+	echoerr "Try '$0 --help' for more information."
+}
+
+export URL_ANTREA_CONFIG_MAP='https://raw.githubusercontent.com/k8snetworkplumbingwg/sriov-network-device-plugin/master/deployments/configMap.yaml'
+export URL_SRIOV_DP_DAEMONSET='https://raw.githubusercontent.com/k8snetworkplumbingwg/sriov-network-device-plugin/master/deployments/k8s-v1.16/sriovdp-daemonset.yaml'
+export URL_WHEREABOUTS_IP_POOLS='https://raw.githubusercontent.com/k8snetworkplumbingwg/whereabouts/master/doc/crds/whereabouts.cni.cncf.io_ippools.yaml'
+export URL_WHEREABOUTS_OVERLAPPING_IP_RANGE_RES='https://raw.githubusercontent.com/k8snetworkplumbingwg/whereabouts/master/doc/crds/whereabouts.cni.cncf.io_overlappingrangeipreservations.yaml'
```

Comment:
maybe a better idea to use a tag rather than `master`?

Reply:
yes, that would be ideal. I will change it to pick from a tag. thanks!